Server Backend/routes/users-route.js

```diff
@@ -1,171 +1,257 @@
 /**
  * Imports the `express`, `router` and other modules
- * for using JWT, cookies
+ * for using JWT, cookies, and axios
  */
 const express = require('express')
 const router = express.Router()
 const jwt = require('jsonwebtoken')
+const Axios = require('axios')
 const config = require('../config')
 const VerifyToken = require('../helpers/VerifyTokens')
 const cookieParser = require('cookie-parser')
+const {URL} = require('url')
 
 router.use(cookieParser())
 
-/**
- * Users array
- */
-var users = []
+const axios = Axios.create({baseURL: 'http://localhost:8091/'})
+
+var error = 0
```

Comment:
Change `var` to `let`.  `var` has global scope while `let` has block scope.

From MDN:
> let allows you to declare variables that are limited in scope to the block, statement, or expression on which it is used. This is unlike the var keyword, which defines a variable globally, or locally to an entire function regardless of block scope.

Reply:
The variable error is used across two helper functions and one endpoint(createUsers) to communicate to the user what kind of error was made. I don't like using globals either but I don't know if using "let" would prevent this functionality altogether.

Reply:
Let me pull the changes in and check changing it to let will affect anything.

Reply:
@cesarbiods From my brief amount of testing, `let` works perfectly fine as well.